### PR TITLE
Fixed rights ref for event hub

### PR DESCRIPTION
**Pull request by MrMCake**
# Change

- Fixed an incorrect default value for event hub deployments 
- Updated test

Pipeline reference
[EventHub: Namespaces](https://github.com/Azure/ResourceModules/actions/workflows/ms.eventhub.namespaces.yml)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

### CODE REVIEW

File: arm/Microsoft.EventHub/namespaces/deploy.bicep [hunk 1]
```diff
@@ -230,13 +230,11 @@ module eventHubNamespace_eventHubs 'eventhubs/deploy.bicep' = [for (eventHub, in
...
       {
         name: 'RootManageSharedAccessKey'
-        properties: {
-          rights: [
-            'Listen'
-            'Manage'
-            'Send'
-          ]
-        }
+        rights: [
```
Comment (eriqua) [inline arm/Microsoft.EventHub/namespaces/deploy.bicep:233]: Are we testing this in the parameters.json?

Comment (MrMCake) [reply arm/Microsoft.EventHub/namespaces/deploy.bicep:233]: Good point. I don't think so. We'd need to add a test case with an empty event hub. Will give it a spin

Comment (MrMCake) [reply arm/Microsoft.EventHub/namespaces/deploy.bicep:233]: done

**Review by rahalan — APPROVED** [review]

**Review by eriqua — APPROVED** [review]
🆗
